.github/workflows/deploy-demo.yml

```
@@ -0,0 +1,43 @@
+        python -m pip install --upgrade pip
+        pip install flet
+        pip install -r requirements.txt
+
+    - name: Build Flet WASM
+      run: |
+        # Publish to 'dist' folder
+        # We point to the main entry point
+        flet publish src/switchcraft/modern_main.py --app-name "SwitchCraft" --app-short-name "SwitchCraft" --app-description "SwitchCraft Web Demo" --base-url "/SwitchCraft/"
+
+    - name: Deploy to GitHub Pages
+      uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.10)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)